modules/auxiliary/admin/sap/cve_2020_6207_solman_rce.rb

```diff
@@ -96,6 +106,75 @@ def report_service_and_vuln
     )
   end
 
+  # Handle incoming HTTP requests from connected agents
+  def on_request_uri(cli, request, script_name)
+    setup_xml_and_variables
```

Comment:
I think `setup_xml_and_variables` has already been called in `action_secstore`. It looks like this method is used to initialize instance variables. Is it required to call it again here?

Reply:
Moved setup_xml_and_variables to `run` method in auxiliary module cve_2020_6207_solman_rce.rb https://github.com/rapid7/metasploit-framework/pull/14965/commits/fa6852d305f926ffb6192950bcabbedd7b423f2f